src/AppBundle/Repository/PostRepository.php

```diff
@@ -38,8 +38,9 @@ public function findLatest($page = 1)
     {
         $query = $this->getEntityManager()
             ->createQuery('
-                SELECT p, t
+                SELECT p, a, t
                 FROM AppBundle:Post p
+                JOIN p.author a
```

Comment:
AFAIK this shouldn't improved nothing, Doctrine hydrates the author without an extra query.